packages/packages-groonga-org-package-task.rb

```diff
@@ -17,10 +17,12 @@
 
 require_relative "../vendor/apache-arrow-source/dev/tasks/linux-packages/package-task"
 require_relative "launchpad-helper"
+require_relative "windows-helper"
```

Comment:
It seems that we don't need to split this to separated file because it's not reusable not like `launchpad-helper`.